pkgs/applications/blockchains/bitcoin.nix

```diff
@@ -58,6 +60,8 @@ stdenv.mkDerivation rec {
   ] ++ optionals (!doCheck) [
     "--disable-tests"
     "--disable-gui-tests"
+] ++ optionals (!withWallet) [
```

Comment:
This looks improperly indented.

Reply:
Indent fixed in 672607c41f28570ea435989ac9f470e528a062ff